definitions/checks/candlepin/validate_candlepin_db.rb

```diff
@@ -0,0 +1,20 @@
+module Checks::Candlepin
+  class ValidateCandlepinDb < ForemanMaintain::Check
+    metadata do
+      description 'Check to validate candlepin database'
+      tags :pre_upgrade
+      label :validate_candlepin_db
+
+      confine do
+        feature(:candlepin) &&
```

Comment:
Why not just confine on `feature(:candlepin_database) && feature(:candlepin_database).check_option_using_cpdb_help('validate')` I would recommend defining the `candlepin` feature to the times that actually has some other reasons. It seems we don't need it at this point.